packages/core/src/llm/model/gateways/gateway-manager.test.ts

```
@@ -0,0 +1,253 @@
+import { describe, expect, it, vi } from 'vitest';
+
+import type {
+  GatewayAuthRequest,
+  GatewayAuthResult,
+  GatewayLanguageModel,
+  MastraModelGatewayInterface,
+  ProviderConfig,
+} from './base';
+import { GatewayManager } from './gateway-manager';
+import { defaultGateways } from './defaults';
```

ESLint

[error] `./defaults` import should occur before import of `./gateway-manager`

(import/order)